lcli/src/main.rs

```diff
@@ -882,6 +883,52 @@ fn main() {
                         .help("Number of repeat runs, useful for benchmarking."),
                 )
         )
+        .subcommand(
+            SubCommand::with_name("mock-el")
+                .about("Creates a mock execution layer server. This is NOT SAFE and should only \
+                be used for testing and development on testnets. Do not use in production. Do not \
+                use on mainnet. It cannot perform validator duties.")
+                .arg(
+                    Arg::with_name("jwt-output-path")
+                        .long("jwt-output-path")
+                        .value_name("PATH")
+                        .takes_value(true)
+                        .required(true)
+                        .help("Path to write the JWT secret."),
+                )
+                .arg(
+                    Arg::with_name("listen-address")
+                        .long("listen-address")
+                        .value_name("IP_ADDRESS")
+                        .takes_value(true)
+                        .help("The server will listen on this address.")
+                        .default_value("127.0.0.1")
+                )
+                .arg(
+                    Arg::with_name("listen-port")
+                        .long("listen-port")
+                        .value_name("PORT")
+                        .takes_value(true)
+                        .help("The server will listen on this port.")
+                        .default_value("8551")
+                )
+                .arg(
+                    Arg::with_name("all-payloads-valid")
+                        .long("all-payloads-valid")
+                        .takes_value(true)
+                        .help("Controls the response to newPayload and forkchoiceUpdated. \
+                            Set to 'true' to return VALID. Set to 'false' to return SYNCING.")
+                        .default_value("false")
+                )
+                .arg(
+                    Arg::with_name("shanghai-time")
+                        .long("shanghai-time")
+                        .value_name("UNIX_TIMESTAMP")
+                        .takes_value(true)
+                        .help("The payload timestamp that enables Shanghai. Defaults to the mainnet value.")
+                        .default_value("1681338479")
```

Comment:
I got this timestamp from the first non-skipped slot after the Capella fork on mainnet: https://beaconcha.in/slot/6209538. I think that's right.